data_gpu/driver/src/data_gpu_top.h

```diff
@@ -42,7 +42,7 @@
 #define PROM_SIZE   0x00050000
 #define USER_OFF    0x00800000
 #define USER_SIZE   0x00800000
-#define GPU_OFF     0x00A00000
+#define GPU_OFF     0x00D00000
```

Comment:
@JJL772 & @mmishra9: Are we going to add a new GPU AXI-Lite interface that's mapped to the AXI-Lite XBAR here to the top-level FW to make sure that the GPU is always on the 0x00D00000 offset:
https://github.com/slaclab/axi-pcie-core/blob/main/shared/rtl/AxiPcieReg.vhd#L112-L168

Reply:
It makes sense to do it. I can make the change to AxiePcieReg.

Reply:
```vhdl
   constant AXI_CROSSBAR_MASTERS_CONFIG_C : AxiLiteCrossbarMasterConfigArray(NUM_AXI_MASTERS_C-1 downto 0) := (
      DMA_INDEX_C     => (
         baseAddr     => x"0000_0000",
         addrBits     => 16,
         connectivity => x"FFFF"),
      PHY_INDEX_C     => (
         baseAddr     => x"0001_0000",
         addrBits     => 16,
         connectivity => x"FFFF"),
      VERSION_INDEX_C => (
         baseAddr     => x"0002_0000",
         addrBits     => 12, -- reduce to 12-bits (that's all that's needed)
         connectivity => x"FFFF"),
      GPU_INDEX_C => (
         baseAddr     => x"0002_8000", -- new offset 
         addrBits     => 12, -- 12-bits (that's all that's needed
         connectivity => x"FFFF"),         
      BPI_INDEX_C     => (
         baseAddr     => x"0003_0000",
         addrBits     => 16,
         connectivity => x"FFFF"),
...
...
...
```

Reply:
In the code above, "#define GPU_OFF 0x00028000" and not at risk of overlapping with the APP address space

Reply:
This seems like a good idea. I had added a crossbar to the InterCardTest firmware to update the offset, would this change remove the need for that?

Reply:
@rclaus2 ping! Just want to keep you in the loop on this

Reply:
https://github.com/slaclab/axi-pcie-core/pull/124

Reply:
https://github.com/slaclab/axi-pcie-core/releases/tag/v5.1.0